scripts/install-snapshot.sh

```diff
@@ -2,7 +2,7 @@
 
 # This script can be used to install/delete snapshotcontroller and snapshot beta CRD
 
-SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"v2.1.1"}
+SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"v3.0.0"}
```

Comment:
oh no, more versions :facepalm: These should really be part of `build.env`

Reply:
I agree with the proposal @nixpanic , will update it. Thanks!